cv/qc-cv.obo

```diff
@@ -940,21 +941,21 @@ name: Peptide sequence column
 def: "The column contains peptide sequences; multiple peptides are separated by space; sequences in ProForma encoding." [PSI:QC]
 comment: ProForma v2 - https://github.com/HUPO-PSI/ProForma/
 is_a: MS:4000107 ! Table column type
-relationship: has_type NCIT:C45253 ! String
+relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
 
 [Term]
 id: MS:4000117
 name: Outliers below column
 def: "The outlier values below 1.5 * IQR." [PSI:QC]
 is_a: MS:4000107 ! Table column type
-relationship: has_type NCIT:C79083 ! Outlier
+relationship: has_units NCIT:C79083 ! Outlier
```

Comment:
This specifies the value of the column, i.e. a column with `Outlier` values. We definitely still need this information in the metric, so don't just remove it. I guess with our renaming of terms to make them consistent with the PSI-MS CV, column metrics are not covered yet, because they're unique to our terms. We should find a good solution for those.

Reply:
The name here should definitely be improved for easier readability and understandability as well. When I read "Outliers below column", my first thought was that there is something below the LC column. Lying on the ground or what? 🤔 Instead, maybe something like "Column containing outlier values"?

Same for the "Outliers above column" term.